### PR TITLE
Build gofer for fedora clients

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: packages/client/gofer/gofer.spec [hunk 1]
```diff
@@ -0,0 +1,1190 @@
...
+%else
+%{py_install}
+%endif
+%endif
+%if 0%{?with_python3}
+%{py3_install}
+%endif
+popd
+
+mkdir -p %{buildroot}/usr/bin
```
Comment (ekohl) [inline packages/client/gofer/gofer.spec]:
```suggestion
mkdir -p %{buildroot}%{_bindir}
```

File: packages/client/gofer/gofer.spec [hunk 2]
```diff
@@ -0,0 +1,1190 @@
...
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}
```
Comment (ekohl) [inline packages/client/gofer/gofer.spec]: `%{_sysconfdir}` already includes `/`:
```suggestion
mkdir -p %{buildroot}%{_sysconfdir}/%{name}
```

File: packages/client/gofer/gofer.spec [hunk 3]
```diff
@@ -0,0 +1,1190 @@
...
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}/plugins
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}/conf.d
+mkdir -p %{buildroot}/%{_sysconfdir}/init.d
+mkdir -p %{buildroot}/%{_sysconfdir}/sysconfig
+mkdir -p %{buildroot}/%{_unitdir}
+mkdir -p %{buildroot}/%{_usr}/lib/%{name}/plugins
+mkdir -p %{buildroot}/%{_usr}/share/%{name}/plugins
```
Comment (ekohl) [inline packages/client/gofer/gofer.spec]:
```suggestion
mkdir -p %{buildroot}%{_datadir}/%{name}/plugins
```

File: packages/client/gofer/gofer.spec [hunk 4]
```diff
@@ -0,0 +1,1190 @@
...
+%if 0%{?with_python3}
+%{py3_install}
+%endif
+popd
+
+mkdir -p %{buildroot}/usr/bin
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}/plugins
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}/conf.d
+mkdir -p %{buildroot}/%{_sysconfdir}/init.d
```
Comment (ekohl) [inline packages/client/gofer/gofer.spec]:
```suggestion
mkdir -p %{buildroot}%{_initddir}
```

**Comment by jlsherrill** [comment]
closing this for now

**Comment by ekohl** [comment]
FYI, I think in the past we manually tagged in gofer from Pulp. Not a great fan of this because I like git to reflect what's in a repo though.

**Pull request closed without merging**